app/_hub/kong-inc/ai-rag-injector/how-to/_index.md

```diff
@@ -4,8 +4,10 @@ title: Using the AI RAG Injector plugin
 ---
 ## Prerequisites
 
-- Create a service and a route
-- Start a [Redis-Stack](https://redis.io/docs/latest/) instance in your environment
+- Create a service and a route.
+- Start a [Redis-Stack](https://redis.io/docs/latest/) instance in your environment. 
+   - Redis-compatible databases (for example, MemoryDB) are not supported
+   - PgVector is supported
```

Comment:
Do we have a definitive list of supported DBs somewhere? I couldn't find one in the docs.

Reply:
We only support pgvector and redis-stack. will add a doc for our vectorDB supported.